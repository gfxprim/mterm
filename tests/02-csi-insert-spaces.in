20 2
hello world\b\b\b\b\b\e[20@
