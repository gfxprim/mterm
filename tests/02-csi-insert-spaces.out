 --------------------
|hello               |
|                    |
 --------------------
size 2x20 cursor 0x6
